linux_os/guide/system/software/integrity/fips/enable_fips_mode/oval/shared.xml

```diff
@@ -2,12 +2,18 @@
   <definition class="compliance" id="{{{ rule_id }}}" version="1">
     {{{ oval_metadata("Check if FIPS mode is enabled on the system") }}}
     <criteria operator="AND">
-      <extend_definition definition_ref="etc_system_fips_exists"
-        comment="check /etc/system-fips file existence"/>
+      <criteria operator="OR">
+        <extend_definition definition_ref="etc_system_fips_exists"
+          comment="check /etc/system-fips file existence"/>
+          <criterion test_ref="test_proc_sys_crypto_fips_enabled"
+            comment="check contents of /proc/sys/crypto/fips_enabled"/>
```

Comment:
Should the check for /proc/sys be used on all systems? Isn't it only for RHEL 10? And should the /etc/system-fips be checked on RHEL 10 if it's deprecated?

Reply:
The `/proc/sys/crypto/fips_enabled` does exist on RHEL 7. I will just use it everywhere.